.github/workflows/publish-latest.yml

```diff
@@ -19,6 +19,11 @@ jobs:
         with:
           # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
           fetch-depth: 0
+      - name: Check if repo has unconsumed changesets
+        id: has-changesets
+        uses: ./.github/actions/has-changesets
+      - name: Echo has-changesets
+        run: echo ${{ steps.has-changesets.outputs.has-changesets }}
```

Comment:
Doesn't do anything to CI yet. Will be followed up by another PR that splits up `publish-latest` using this action.